README.md

```diff
@@ -0,0 +1,15 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version - 2.6.8p205
+* Rails version 5.2.8
+* run 'rails g scaffold work_orders technician_id:string location_id:string time:datetime duration:integer price:integer'
+* run 'rails g scaffold locations name:string city:string'
+* run 'rails g scaffold technicians name:string'
```

Comment:
These are not instructions for how to get the project set up. They are steps you may have used during development but do not / cannot be run again. 

You would probably want to list something like:

```rails db:create```
```rails db:migrate```

here.